docs/topics/simulation_setup.rst

```diff
@@ -0,0 +1,182 @@
+<https://doi.org/10.1063/1.3601919>`_.  These have been implemented for OPS,
+and will be added to the core code in the future.
+
+-----
+
+Getting an initial trajectory
+=============================
+
+Even with configurational sampling, initial conditions are not always
+completely trivial. A PDB entry might be missing hydrogens, and probably
+needs to be solvated (and energy-minimized). However, obtaining initial
+conditions for path sampling of rare events is much more difficult. Path
+sampling makes it possible to generate many rare trajectories once a first
+rare trajectory is given as input. But how can you get that first rare
+trajectory?
+
+The answer is, "any way that works." Some approaches include running long
+MD, or using a ratcheting approach (such as, or similar to, adaptive
+multilevel splitting) to get natural trajectories that go further and
+further toward the products.
+
+Another important idea is that the initial trajectory does not need to be a
+valid physical trajectory from the dynamics of this path ensemble. For
+example, it can come from a run at a different temperature. A non-physical
+path can be equilibrated to a physical path by performing an equilibration
+path sampling simulation before performing the production simulation.
+
+Note that a very bad initial trajectory could equilibrate to a nonsensical
+region of trajectory space, just like a bad configuration could equilibrate
+to a nonsensical metastable state in configurational Monte Carlo. Selecting
+a "nearly" physical path is important.
+
+One approach that has shown promise is to use metadynamics to obtain a first
+transition. After the first transition, metadynamics usually will not have
+significantly altered the potential energy surface near the barrier, so this
+trajectory is likely to equilibrate to a good initial trajectory.
+
+Generating with OPS
+-------------------
+
+There are a few approaches within OPS for generating initial trajectories.
+One approach is to use run with a higher temperature. This is used in the
+alanine dipeptide examples, such as the notebooks for :ref:`AD-tps`. This
+can work in practical cases for transitions that are not too rare.
+
+Another approach is to use the :class:`.FullBootstrapping` approach, which
+starts from a snapshot and rachets up through the path ensembles for a given
+transition. This will sample each ensemble until the trajectory satisfies
+the next ensemble, then it switches to the new ensemble and samples that.
+The process continues until the all ensembles in the transition have
+trajectories. It is essentially a version of adaptive multilevel splitting
+that has been discretized along the progress parameter. This approach sounds
+promising, but in reality is very dependent on the quality of the order
+parameter. It is very efficient some simple models, which is why we use it
+for the examples like the :ref:`toy-mstis` notebooks. In complicated
+systems, it may fail.
+
+
```

Comment:
As pointed out above, a committor calculation can help. The committor calculation is available in OPS